courses/free-courses-en.md

```diff
@@ -1331,6 +1325,7 @@
 
 ### PHP
 
+* [Laravel 9 Tutorial](https://www.youtube.com/playlist?list=PL8p2I9GklV47Jszga434vZxOmY74Q1N_K) - Anil Sidhu (Code Step By Step)
```

Comment:
🤔 I am wondering if the linter should have picked this up and moved it down a line? Not sure if numbers are considered in the alphabetization.

Reply:
Yeah, the linted did pick it up and suggested I move it down a line. It is now located under the Laravel 5.something tutorial,